csharp/extractor/Semmle.Extraction.CSharp/Extractor/ExtractionContext.cs

```diff
@@ -15,6 +15,7 @@ public class ExtractionContext
         public ExtractorMode Mode { get; }
         public string OutputPath { get; }
         public IEnumerable<CompilationInfo> CompilationInfos { get; }
+        public bool IsStandalone => Mode.HasFlag(ExtractorMode.Standalone);
```

Comment:
Do we still need a public `Mode` property? Can it be made private?

Reply:
No, it the value itself is written to trap outside this file (it is written to `file_extraction_mode`).